LoopTests/Mock Stores/HKHealthStoreMock.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  HKHealthStoreMock.swift
```

Comment:
I pulled this in so an actual (non-mock) carb store could be used to calculate carb effects for the missed meal detection unit tests instead of loading the carb effects via fixtures